src/footer.js

```diff
@@ -0,0 +1,31 @@
+class Footer extends HTMLElement {
+    constructor() {
+      super();
+    }
+  
+    connectedCallback() {
+      this.innerHTML = `
+        <footer>
+          <ul class="side-map">
+            <li><a class="a-footer" href="about.html">About</a></li>
+            <li><a class="a-footer" href="terms.html">Terms of use</a></li>
+            <li><a class="a-footer" href="contact.html">Contact</a></li>
+          </ul>
+          <ul class="social-row">
```

Comment:
jeśli chodzi o design to może przeneśmy te 2 zdjęcia na środek footera.
Myślę że możemy je delikatnie wyszarzyć i jak zrobisz hovera to odzyskują naturalne barwy

Reply:
to podkreślenie też możesz wywalić

Reply:
Przeniesione i hover przerobiony na zmianę opacity. Podkreślenie wywalone.